Formula/tinker.rb

```diff
@@ -38,7 +38,6 @@ class Tinker < Formula
   
   # Runtime dependencies that will be used by Tinker.
   depends_on "awscli" => "2"
-  depends_on "snapsheet/core/session-manager-plugin" => "1.2.295"
```

Comment:
Tinker does not yet user this, and may not need it if we use the Hackday alternative.